spec/helpers/feature_flag_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "rails_helper"
+
+UserStruct = Struct.new(:flipper_id)
+
+describe FeatureFlag, type: :helper do
+  describe ".strict_enabled?" do
+    it "calls Flipper's enabled? method" do
+      allow(Flipper).to receive(:enabled?).with("foo")
+
+      described_class.strict_enabled?("foo")
+
+      expect(Flipper).to have_received(:enabled?).with("foo")
+    end
+  end
+
+  describe ".exist?" do
+    it "calls Flipper's exist? method" do
+      allow(Flipper).to receive(:exist?).with("foo")
+
+      described_class.exist?("foo")
+
+      expect(Flipper).to have_received(:exist?).with("foo")
+    end
+  end
+
+  describe ".enabled?" do
+    let(:user) { UserStruct.new(flipper_id: 1) }
+
+    context "when flag doesn't exist" do
+      it "returns true" do
+        expect(described_class.enabled?("missing_flag")).to be_truthy # rubocop:disable Rspec/PredicateMatcher
```

Comment:
If we return a truthy value for pages where we don't have feature flags defined, wouldn't that disable all existing pages unless we retroactively add flags for them?